airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1458,6 +1479,15 @@ components:
       properties:
         connectionId:
           $ref: "#/components/schemas/ConnectionId"
+    WebBackendConnectionIdRequestBody:
```

Comment:
`WebBackendConnectionIdRequestBody` => `WebBackendConnectionRequestBody`? maybe? it's not just an id anymore.